src/app/views/marketplace/marketplace.controller.coffee

```diff
@@ -1,8 +1,12 @@
 angular.module 'mnoEnterpriseAngular'
-  .controller('DashboardMarketplaceCtrl', (MnoeMarketplace) ->
+  .service('greeting', ->
+    greeting = this
+    greeting.message = 'Default')
```

Comment:
Please put this closing bracket on a newline, to keep it the same as the rest of the files in the project